sql/core/src/main/scala/org/apache/spark/sql/SparkSQLParser.scala

```diff
@@ -57,6 +57,7 @@ private[sql] class SparkSQLParser(fallback: String => LogicalPlan) extends Abstr
   protected val AS = Keyword("AS")
   protected val CACHE = Keyword("CACHE")
   protected val CLEAR = Keyword("CLEAR")
+  protected val CONF = Keyword("CONF")
```

Comment:
I saw the following comments in the `SetCommand`:
```
    // Queries all key-value pairs that are set in the SQLConf of the sqlContext.
    // Notice that different from Hive, here "SET -v" is an alias of "SET".
    // (In Hive, "SET" returns all changed properties while "SET -v" returns all properties.)
    case Some(("-v", None)) | None =>
      sqlContext.getAllConfs.map { case (k, v) => Row(s"$k=$v") }.toSeq
```
How about using `SET -v` to display all configurations?